### PR TITLE
refactoring to range loop

### PR DESCRIPTION
**Pull request by Joilnen**

### CODE REVIEW

File: OgreMain/src/OgreCamera.cpp [hunk 1]
```diff
@@ -174,7 +171,7 @@ namespace Ogre {
 
         Quaternion targetWorldOrientation;
 
-        if( mYawFixed )
```
Comment (paroj) [inline OgreMain/src/OgreCamera.cpp:177]: I would prefer not to do these whitespace changes manually. They are trivial to do with clang-format. I just refrained to apply them as such commit introduces lots of noise in the code (think of open pull requests) without adding much.

Comment (Joilnen) [reply OgreMain/src/OgreCamera.cpp:177]: is there a .clang-format ready to use somewhere or we can use llvm with 4 space of indentation ?

Comment (paroj) [reply OgreMain/src/OgreCamera.cpp:177]: added the clang-format I use to #2584. But still, I do not think we should apply it to existing code just like that..

Comment (Joilnen) [reply OgreMain/src/OgreCamera.cpp:177]: applied a slightly similar format than your, I think it looks good

**Review by paroj — CHANGES_REQUESTED** [review]
like I said initially, i dont think it make sense to format existing code just for the sake of it..

**Comment by Joilnen** [comment]
Ok I got if that keeps similar the actual style would be worth to make it, anyway I am reverting or creating a new commit.

**Pull request merged**